packages/block-library/src/navigation/frontend.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * External dependencies
+ */
+import MicroModal from 'micromodal';
+
+window.addEventListener( 'DOMContentLoaded', ( event ) => {
+	MicroModal.init();
+} );
```

Comment:
This triggers the `@wordpress/no-global-event-listener` rule — thinking about disabling as this is supposed to run directly in the browser. 

Otherwise, we'll have to find an arbitrary element from which to pull `ownerDocument` from.

Reply:
You could also use `@wordpress/dom-ready` that is used in other places:
https://github.com/WordPress/gutenberg/tree/trunk/packages/dom-ready